.rubocop.yml

```diff
@@ -154,7 +154,6 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
```

Comment:
Why was this removed?

Reply:
It has been deprecated by Rubocop. This cop now automatically inherits the max line length from the `Metrics/LineLength` cop.